<ui version="4.0" >
 <class>xmppWin</class>
 <widget class="QMainWindow" name="xmppWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>451</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Kapture [Offline]</string>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QComboBox" name="profilesComboBox" />
      </item>
      <item>
       <widget class="QComboBox" name="statusBox" >
        <item>
         <property name="text" >
          <string>Online</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Free for chat</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Do not disturb</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Away</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Far far away</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Invisible</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Offline</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tlsIconLabel" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Normal View</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="configBtn" >
            <property name="text" >
             <string>Configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="addItemBtn" >
            <property name="toolTip" >
             <string>Add a contact</string>
            </property>
            <property name="text" >
             <string>+</string>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MouseRosterTableView" native="1" name="tableView" >
          <property name="showGrid" stdset="0" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Graphic View</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Graphic View is not to be used for now.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>If you do so, please use it carefully and with a testing account only.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView" >
          <property name="windowModality" >
           <enum>Qt::NonModal</enum>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseRosterTableView</class>
   <extends>QWidget</extends>
   <header>mouserostertableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
